docs/guides/deploy-svm-warp-route.mdx

```diff
@@ -0,0 +1,145 @@
+
+:::info
+**If you want your SVM rollup supported as a core Hyperlane deployment, or are looking to set up an EVM to SVM Warp Route, [get in touch](https://forms.gle/KyRTaWvo4XNmNvrq6)!**
+:::
+
+## Warp Route Types
+
+The type of token used determines the Warp Route type, so it's important to understand the different Warp Route contracts available:
+- [Native](https://docs.hyperlane.xyz/docs/protocol/warp-routes/warp-routes-types#native-token-warp-routes): Handles the transfer of native gas tokens (e.g. SOL on Solana, ETH on Eclipse).
+- [Collateral](https://docs.hyperlane.xyz/docs/protocol/warp-routes/warp-routes-types#collateral-backed-erc20-warp-routes): Handles the transfer of existing [Token-2022](https://spl.solana.com/token-2022) tokens (the ERC20 equivalent on SVM).
+- [Synthetic](https://docs.hyperlane.xyz/docs/protocol/warp-routes/warp-routes-types#synthetic-erc20-warp-routes): Handles synthetic tokens that are minted and burned as transfers occur through the Warp Route, to represent tokens from their origin chain. The tooling in this guide deploys a new Token-2022 token in this case, whose authority is set to the deployer key.
+
+Here are the common Warp Route setups (you can find more details [here](https://docs.hyperlane.xyz/docs/protocol/warp-routes/warp-routes-example-usage)):
+- Native to Native: Transfer a token that is native to both connected chains, such as ETH between Optimism and Arbitrum. Rebalancing liquidity is an important consideration in this case.
+- Native to Synthetic: Lock Native tokens on the origin chain to mint Synthetic ones on the destination. When transferring back, the Synthetic is burned. An example of this is a SOL Warp Route between Solana and Eclipse.
+- Collateral to Synthetic: Lock Collateral tokens on the origin chain to mint Synthetic ones on the destination. When transferring back, the Synthetic is burned. An example of this is a USDC Warp Route between Solana and Eclipse.
+- Other: Collateral to Collateral is also possible, if the token already exists on both the origin and destination chains. Rebalancing liquidity is an important consideration in this case.
+
+## Before You Start
+
+Deploying a Warp Route requires there to be a core Hyperlane deployment that is connected (i.e. actively relayed and secured) to the rest of the Hyperlane ecosystem. The core Hyperlane deployments used in this guide are Solana ([core artifacts](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/sealevel/environments/mainnet3/solanamainnet/core/program-ids.json)) and Eclipse ([core artifacts](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/sealevel/environments/mainnet3/eclipsemainnet/core/program-ids.json)). You may need to refer to these core artifacts throughout the guide.
+
+
+## Deploy a Sealevel Warp Route
+
+1. Install `solana-cli 1.14.20` to build the Warp Route programs:
```

Comment:
I think this is what I do below, on line 49. Let me know how it could come across more clearly

Reply:
Yeah you do call it out, so I don't feel too strongly about changing

The reason I called it out is I could still see someone not totally getting that these specific versions are strictly required. So was suggesting more strongly wording it like "Using this specific version is required for XXX otherwise you'll experience undefined behavior" or something

Reply:
good call, got it. Just updated